### DISCUSSION TITLE
Toastify now working in nextjs

### DISCUSSION THREAD

**Discussion by cwik-tech** [fkhadra/react-toastify#883] [Q&A]
I am porting my app from react 18 to nextjs 13.0.6 (with app folder) and toastify does not seem to work. It does not show any error but it does not simply work. I presume I did not place the ToastContainer properly, but I am not sure. could someone help? This is my _app and layout folders:
![image](https://user-images.githubusercontent.com/54542292/205529741-272ef7e0-66b0-4493-9a66-f002fbf02aa0.png)

**Comment by vnugent** [comment]
I'm having the same issue with next 12.x.  I've also placed ToastContainer in _app.tsx and nothing is showing up. 

Edited: you need to import the css file

```javascript
// _app.tsx
import { ToastContainer } from 'react-toastify'
import 'react-toastify/dist/ReactToastify.css'



```
